week-3-practise/level-1/3-filter.js

```diff
@@ -545,7 +545,11 @@ var streetNames = [
   "Wrestler's Court"
 ];
 
-var laneNames; // Complete this line
+function nameOfStreet ( name) {
+  return name === "lane" ;
+}
+
+var laneNames = streetNames.filter(nameOfStreet);
```

Comment:
Does this give the expected output?

Reply:
Yes it does give the right results.

Reply:
Are you sure? :slightly_smiling_face: 
If I run this code I get:

```
$ node lanes.js
[]
0
```

Reply:
It looks like you opened another PR on a different branch, so that's why it isn't showing here.

Reply:
Yes you are right it is under a different branch with the name week-3-practise